src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const [itemFreq, setItemFreq] = useState(7);
+  const [formValidationVisual, setFormValidationVisual] = useState(false);
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(capitalize(event.target.value));
+    setFormValidationVisual(true);
+  };
+
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: itemFreq,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+    setGroceryItem('');
+    setItemFreq(7);
+    setFormValidationVisual(false);
```

Comment:
It seems like this reset of the form isn't working `onSubmit`. This is after I've submitted:

![image](https://user-images.githubusercontent.com/10378331/114868024-17486e00-9dbb-11eb-8670-4207fa525863.png)